server/src/main/java/io/crate/execution/engine/FirstColumnConsumers.java

```diff
@@ -55,6 +57,7 @@ public Supplier<List<Object>> supplier() {
         @Override
         public BiConsumer<List<Object>, Row> accumulator() {
             return (agg, row) -> {
+                rowAccounting.accountForAndMaybeBreak(row);
```

Comment:
Given that this is always a single value it might be worth checking if going with `RamAccounting` and `dataType.valueBytes(value)` would have less overhead.

`RowAccounting` occassionally pops up in profiles, and involves a small chain of abstractions.
We also had cases where a `Row.get()` call evaluates a function (E.g. via `InputRow`) and then doing that call twice results in duplicate evaluations.